justfile

```diff
@@ -17,6 +17,11 @@ set positional-arguments
     @cargo build --features debug
     ./target/debug/clementine $1
 
+# run clemente in DEBUG mode
+@test_bitmap *args='':
+    @cargo build --features test_bitmap
+    ./target/debug/clementine $1
```

Comment:
You can simply run `cargo run --features test_bitmap -- $1`

Reply:
True. I don't know why I did this ahah